topics/ecology/tutorials/species-distribution-modeling/tutorial.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: tutorial_hands_on
+topic_name: sdm
+tutorial_name: species distribution modeling
+---
+
+# Introduction
+{:.no_toc}
+
+Species Distribution Modeling (SDM) can help understand the distribution of a species depending on its environment. It can also attempt to quantify the impact of climate change on the species habitat, direct conservation efforts and predict invasive species distributions. This is done by associating data of species occurences (observations) with a set of environmental data (such as temperature and precipitation).
+
+The goal of this tutorial is to model a theorical ecological niche and predict species distribution in a future climate scneario by using SDM with the Wallace interactive environment on Galaxy. We'll use the data occurrences of US *Chrysemys picta* ([Painted turtle](https://en.wikipedia.org/wiki/Painted_turtle)) from the North America region.
+
+# Step 1: Loading a dataset
+
+In this study the datasets are all imported from the tool `Get species occurrences data and taxref informations` in Galaxy-E. With this tool, data are available from differents databanks like [GBIF](https://www.gbif.org/), [bison](https://www.gbif.org/), [iNaturalist](https://www.inaturalist.org/) and others.
+
+>    > ### {% icon tip %} Tip: Importing data set from a data bank
```

Comment:
You do not need the `>    ` at the beginning of the lines there and below

Reply:
done